musicrs/util/slack_api.py

```diff
@@ -0,0 +1,44 @@
+"""
+Slack API utility
+"""
+import os
+import json
+import slack
+import dotenv
+
+dotenv.load_dotenv()
```

Comment:
Let's load all env file from settings.py. You can take reference from msdf repo

Reply:
Done